go/inst/instance_dao.go

```diff
@@ -3068,6 +3155,79 @@ func FigureInstanceKey(instanceKey *InstanceKey, thisInstanceKey *InstanceKey) (
 	return figuredKey, nil
 }
 
+// Obtain information about Group Replication  for this host as well as other hosts who are members of the same group
+// (if any). This method swallows up any errors, interpreting them as if the host does not support group replication
+// (e.g. because the group replication plugin is not loaded, therefore the corresponding performance schema tables and
+// global variables are not known to the server.
+func PopulateGroupReplicationInformation(instance *Instance, db *sql.DB) {
+	q := `
+	SELECT
+		MEMBER_ID,
+		MEMBER_HOST,
+		MEMBER_PORT,
+		MEMBER_STATE, 
+		MEMBER_ROLE,
+		@@global.group_replication_group_name,
+		@@global.group_replication_single_primary_mode
+  	FROM
+		performance_schema.replication_group_members
+  	 `
+	rows, err := db.Query(q)
```

Comment:
why not `db.QueryRow`? https://golang.org/pkg/database/sql/#example_DB_QueryRowContext 
 I think is being used everywhere already, otherwise please add `defer rows.Close()`